src/style.css

```diff
@@ -1,3 +1,89 @@
+html {
+  margin: 0;
+  display: block;
+
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
+  display: block;
```

Comment:
Similar to the `html` selector, the `display: block;` is not needed on the `body` tag as it is the default display value.